lib/find.coffee

```diff
@@ -135,10 +135,18 @@ module.exports =
   createViews: ->
     return if @findView?
 
-    history = {@findHistory, @replaceHistory, @pathsHistory}
+    findBuffer = new TextEditor(mini: true).getBuffer()
```

Comment:
Yeah. I wanted them to be consistent in the way they created the editors between the views. Looks like `TextBuffer` is exported. I can just create them directly.